hudi-common/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderBase.java

```diff
@@ -276,6 +277,10 @@ private void validateOutputFromFileGroupReader(StorageConfiguration<?> storageCo
     props.setProperty("hoodie.payload.ordering.field", "timestamp");
     props.setProperty(RECORD_MERGER_STRATEGY.key(), RECORD_MERGER_STRATEGY.defaultValue());
     props.setProperty(RECORD_MERGE_MODE.key(), recordMergeMode.name());
+    props.setProperty(HoodieMemoryConfig.MAX_MEMORY_FOR_MERGE.key(), String.valueOf(HoodieMemoryConfig.MAX_MEMORY_FOR_MERGE.defaultValue()));
```

Comment:
`1024 * 1024 * 1000 ` replace defaultValue

Reply:
@codope asked me to do this in his comments 
https://github.com/apache/hudi/pull/11455#discussion_r1638968211
https://github.com/apache/hudi/pull/11455#discussion_r1638968494